### PR TITLE
Migrate to Java 21

### PR DESCRIPTION
**Pull request by ravinperera00**
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: native/src/main/java/io/ballerina/lib/soap/Util.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+/**
+ * Util methods for SOAP operations.
+ */
+public class Util {
+    private Util() {
+        //private constructor
+    }
+
+    public static void notifyFailure(BError bError) {
```
Comment (warunalakshitha) [inline native/src/main/java/io/ballerina/lib/soap/Util.java:31]: Do we need separate class here. we can simple use bError.printStackTrace();

Comment (warunalakshitha) [reply native/src/main/java/io/ballerina/lib/soap/Util.java:31]: Need to add  System.exit(1);

Comment (ravinperera00) [reply native/src/main/java/io/ballerina/lib/soap/Util.java:31]: I'll fix that.

Comment (ravinperera00) [reply native/src/main/java/io/ballerina/lib/soap/Util.java]: `System.exit(1)` gives a spotbugs error which is why I removed it.

Comment (ravinperera00) [reply native/src/main/java/io/ballerina/lib/soap/Util.java]: Fixed with 752f2efd1511410cd25dd5d6a3131794db216c53

**Review by warunalakshitha — APPROVED** [review]
LGTM

**Pull request merged**